app/components/ListClaimConditions/ListClaimConditions.tsx

```
@@ -0,0 +1,286 @@
+            variant="outline"
+            className="bottom-1 border border-slate-400 p-2 text-sm font-medium"
+            colorScheme={props.addClaimPhase ? 'red' : ''}
+            leftIcon={
+              !props.addClaimPhase ? (
+                <AddIcon fontSize={10} />
+              ) : (
+                <CloseIcon fontSize={10} />
+              )
+            }
+            onClick={() => {
+              props.setAddClaimPhase!(!props.addClaimPhase);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)